pennathletics/athletes.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import print_function #PYTHON 3 BABY
+import models, parse
+import scraper as s
+
+def getRoster(sport, year):
+    roster = []
+    raw = s.scrapeRoster(sport, year)
+    for person in raw:
+        roster.append(
+            models.Athlete(
+            person[0],
+            person[1],
+            person[2],
+            parse.ht_feet_to_inches(person[3]),
+            person[4],
+            person[5],
+            parse.parse_hometown(person[6]))
```

Comment:
Got this error when running `>>> athletes.get_player('W_Basketball', 2015, no = 0)`

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "pennathletics/athletes.py", line 25, in get_player
    potential_list = get_roster(sport, year)
  File "pennathletics/athletes.py", line 17, in get_roster
    parse.parse_hometown(person[6]))
IndexError: list index out of range
```